feature/qos/otg_tests/ingress_police_two_rate_three_color/README.md

```diff
@@ -0,0 +1,221 @@
+# DP-2.5 Police traffic on input matching all packets using 2 rate, 3 color marker
+
+## Summary
+
+Use IP address and mac-address from TE-18.1 gRIBI. Static Routes can be used for this.
```

Comment:
Can you please just define this topology in this file?

The TE-18.1 reference here is problematic because:

a) TE-18.1 says "TODO: Complete test environment setup steps"
b) Some of the things you refer to (like `dest_A`) are not actually defined there.